documentation/operators/src/nodes/network-requester-setup.md

```diff
@@ -164,11 +164,11 @@ Now that we have initialized our network-requester, we can start it with the fol
  ./nym-network-requester run --id <YOUR_ID>
 ```
 
-## Using your network requester
+## Using your Network Requester
 
-The next thing to do is use your requester, share its address with friends (or whoever you want to help privacy-enhance their app traffic). Is this safe to do? If it was an open proxy, this would be unsafe, because any Nym user could make network requests to any system on the internet.
+The next thing to do is use your requester, share its address with friends (or whoever you want to help privacy-enhance their app traffic). Is this safe to do? If it was an open proxy, this would be unsafe, because any Nym user could make Network requests to any system on the internet.
```

Comment:
'Network requests' -> 'network requests', no need to capitalise here

Reply:
brain fog, will fix

Reply:
done